Gemfile

```diff
@@ -31,6 +31,7 @@ gem 'sass-rails', '~> 5.0'
 gem 'secure_headers', '~> 3.0.0'
 gem 'sidekiq'
 gem 'simple_form', github: 'amoose/simple_form', branch: 'feature/aria-invalid'
+gem 'sinatra', require: false
```

Comment:
Was there a particular reason you moved sinatra to the main group? As far as I know, it's only needed for tests (to run the fake IdPaaS server).